util/src/zip.rs

```diff
@@ -37,7 +52,7 @@ pub fn create_zip(dst_file: &File, src_dir: &Path, files: Vec<PathBuf>) -> io::R
 		let file_path = src_dir.join(x);
 		if let Ok(file) = File::open(file_path.clone()) {
 			info!("compress: {:?} -> {:?}", file_path, x);
-			writer.get_mut().start_file_from_path(x, options)?;
+			writer.get_mut().start_file(path_to_string(x), options)?;
```

Comment:
Would just `x.to_str()` work here? We generate the path, so it shouldn't need to be sanitized with the `path_to_string` function, right?

Reply:
`path_to_string` performs extra checks that `x.to_str()` does not, like normalizing the path.

Reply:
:+1: